library/ssl_tls.c

```diff
@@ -2872,6 +2866,8 @@ int ssl_parse_certificate( ssl_context *ssl )
             if( pk_can_do( pk, POLARSSL_PK_ECKEY ) &&
                 ! ssl_curve_is_acceptable( ssl, pk_ec( *pk )->grp.id ) )
             {
+                ssl->session_negotiate->verify_result |= BADCERT_BAD_EC;
```

Comment:
The failure verification code here is defined as:
```
#define BADCERT_BAD_EC            0x4000
```
However, the failure value in the PR to development is:
```
#define MBEDTLS_X509_BADCERT_BAD_KEY         0x010000
```
Could you please explain why these are different?